### ISSUE TITLE
Select fields missing labels in CMS

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#10132]
## Description
In the CMS on VAMC System Operating Status content type, there are select fields displaying the sort order for each row of the tables but these select fields do not have associated labels.

## Additional Context
This is also occurring on the VAMC Facility Health Service content type and VAMC Facility Non-clinical Service content type.

## Screenshot
![image](https://user-images.githubusercontent.com/106678594/183916909-11d02179-3a6f-4420-9ade-70a4c545132a.png)

## Accessibility Standard

WCAG version 2.0 A, [Criterion 3.3.2](https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html)

## Acceptance Criteria
- [ ] Technical review to determine feasibility of adding labels
- [ ] determine display of labels
- [ ] Change Management consulted
- [ ] Implementation ticket created

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [x] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`
